sql/postgres/sqlspec.go

```diff
@@ -65,12 +66,15 @@ func MarshalSpec(v interface{}, marshaler schemaspec.Marshaler) ([]byte, error)
 	return specutil.Marshal(v, marshaler, schemaSpec)
 }
 
-// Realm converts the schemas and tables into a schema.Realm.
-func Realm(schemas []*sqlspec.Schema, tables []*sqlspec.Table) (*schema.Realm, error) {
+// Realm converts the schemas and tables of the doc into a schema.Realm.
+func Realm(d doc) (*schema.Realm, error) {
```

Comment:
I prefer not to use an unexported type as input to an exported method

Reply:
True